portfolio/src/main/webapp/index.html

```diff
@@ -27,6 +27,7 @@ <h1>Sara's Portfolio</h1>
         <br><br>
         <input type="submit">
       </form>
+      <div id="comment-data"></div>
```

Comment:
Are there additional javascript changes required? Is that coming in a separate PR?

Reply:
None in the requirements, changes to come when I take care of styling